tests/common/factories.py

```diff
@@ -104,6 +104,7 @@ class Meta:  # pylint: disable=no-init, old-style-class
     target_uri = factory.Faker('uri')
     text = factory.Faker('paragraph')
     userid = factory.Faker('user_name')
+    document = factory.SubFactory(Document)
```

Comment:
Is there a reason we're not importing the `Annotation` and `Document` factories from `tests.memex.factories` rather than duplicating them?